README.md

```diff
@@ -71,18 +71,14 @@ question. If that doesn't work, try opening an [issue] with the question.
 
 ## Project Layout
 
-- [`tonic`](https://github.com/hyperium/tonic/tree/master/tonic): Generic gRPC and HTTP/2 client/server
-implementation.
-- [`tonic-build`](https://github.com/hyperium/tonic/tree/master/tonic-build): [`prost`] based service codegen.
-- [`tonic-types`](https://github.com/hyperium/tonic/tree/master/tonic-types): [`prost`] based grpc utility types
-  including support for gRPC Well Known Types.
-- [`tonic-health`](https://github.com/hyperium/tonic/tree/master/tonic-health): Implementation of the standard [gRPC
-health checking service][healthcheck]. Also serves as an example of both unary and response streaming.
-- [`tonic-reflection`](https://github.com/hyperium/tonic/tree/master/tonic-reflection): A tonic based gRPC
-reflection implementation.
-- [`examples`](https://github.com/hyperium/tonic/tree/master/examples): Example gRPC implementations showing off
-tls, load balancing and bi-directional streaming.
-- [`interop`](https://github.com/hyperium/tonic/tree/master/interop): Interop tests implementation.
+- [`tonic`](./tonic): Generic gRPC and HTTP/2 client/server implementation.
```

Comment:
Do you think this works in the context of a README displayed on crates.io?

Reply:
I think this works like the link to `LICENSE`.

Reply:
Okay, fair.